scripts/agent-et-motif-changent-de-service.rb

```diff
@@ -0,0 +1,45 @@
+# rails runner scripts/agent-et-motif-changent-de-service.rb
+
+ORGANISATION_ID = 1
+IDS_SERVICE_SOURCE = [1, 2]
+ID_SERVICE_DESTINATION = 4
+
+services_source = Service.where(id: IDS_SERVICE_SOURCE)
+service_destination = Service.find(ID_SERVICE_DESTINATION)
+
+puts "Hello !"
+puts "Déplace les agents et les motifs des services #{services_source.map(&:name).join(", ")} vers le service #{service_destination.name}"
+
+Agent.transaction do
+  Motif.transaction do
```

Comment:
Je crois que ça ne sert à rien de nester les transactions ici, non? Si je dis pas de bêtise, la méthode `transaction` est en fait globale à la connection à la base, et ne dépend pas du modèle (ou de la table). En fait on pourrait écrire un seul `ApplicationRecord.transaction` et ça reviendrait au même.

(J’ai repêché [la doc](https://api.rubyonrails.org/classes/ActiveRecord/Transactions/ClassMethods.html), qui dit:
> Though the transaction class method is called on some Active Record class, the objects within the transaction block need not all be instances of that class. This is because transactions are per-database connection, not per-model.

Reply:
J'ai eu un doute lors d'un test... je remet un seul niveau.